### PR TITLE
readme

### PR DESCRIPTION
**Pull request by yegor256**
Updating readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,18 @@ The following set of features makes Artipie unique among all others:
...
+    [Docker](./examples/docker),
+    [Rubygems](./examples/gem),
+    [Go](./examples/go),
+    [Helm](./examples/helm),
+    [Npm](./examples/npm),
+    [NuGet](./examples/nuget),
+    [Composer](./examples/php),
+    [Pip](./examples/python),
+    [Rpm](./examples/rpm),
+    and [others](./examples)
```
Comment (Sammers21) [inline README.md:36]: Those others are: python and maven

Comment (yegor256) [reply README.md:36]: @Sammers21 can be anything. It's just a placeholder for the future.

**Comment by 0crat** [comment]
Job was finished in 6 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request merged**